peer/roundrobin/peerring.go

```diff
@@ -53,7 +53,7 @@ type peerRing struct {
 
 // Add a peer.StatusPeer to the end of the peerRing, if the ring is empty it
 // initializes the nextNode marker
-func (pr *peerRing) Add(p peer.StatusPeer) peer.Subscriber {
+func (pr *peerRing) Add(p peer.StatusPeer, _ peer.Identifier) peer.Subscriber {
```

Comment:
Similar to `pendingHeap`, let's add a compile-time check here.

Reply:
✔